lib/utils.js

```diff
@@ -505,8 +505,30 @@ function normalizePlatformVersion (originalVersion) {
   return `${major}.${minor}`;
 }
 
+/**
+ * Takes a desired capability and tries to JSON.parse it as an array,
+ * and either returns the parsed array or a singleton array.
+ *
+ * @param {any} cap A desired capability
+ */
+function parseArray (cap) {
```

Comment:
This should all exist in `appium-base-driver` so that it can be used here and in `appium-android-driver`.

Reply:
Sounds good. Should I move the `otherApps` capability too?

Reply:
If it can work with pre-XCUITest apps, then yes. Otherwise it should be split up in the two places, unfortunately.

Reply:
Ok, one thing at a time. I'll leave it here for now.

Reply:
@imurchie https://github.com/appium/appium-base-driver/pull/333